utils/logger.go

```diff
@@ -0,0 +1,21 @@
+package utils
+
+import (
+	log "github.com/sirupsen/logrus"
+	"github.com/xanzy/go-gitlab"
+)
+
+func Logger(project *gitlab.Project, mergeRequest *gitlab.MergeRequest) *log.Entry {
+	fields := log.Fields{
+		"project": project.NameWithNamespace,
+		"webURL":  project.WebURL,
+	}
+	if mergeRequest != nil {
+		fields = log.Fields{
+			"project":      project.NameWithNamespace,
+			"mergeRequest": mergeRequest.ID,
```

Comment:
@mariusheine These are redundant.

Reply:
@lukashass  Why do you think they are redundant? Because I also printed the webURL? Or does the merge request id contain also the project info?

Reply:
@lukashass Or do we need to use the IID fields always :confused:

Reply:
@mariusheine Yes I think the URL is sufficient.

Reply:
@lukashass OK we must definitely use the IID field of the merge request but I still don't see the redundancy between the project name with namespace and the merge request id. 

Despite the weburl. But this is hard to interpret in a fast way but totally useful when we want to directly open the merge request/project from within the logs.

A log would then look like 

`time="2009-11-10T23:00:00Z" level=info msg="Test log" mergeRequest=iid project="namespace / project" webURL="https://gitlab.com/namespace/project/-/merge_requests/iid"` 

when you execute the following 

```golang
log.WithFields(log.Fields{
			"project":      project.NameWithNamespace,
			"mergeRequest": mergeRequest.ID,
			"webURL":       mergeRequest.WebURL,
		}).Info("Test log")
```

Reply:
@lukashass Alright. Then lets use the URL simply.

Reply:
Changed it to only log the webURL